src/ir/selector_access_path.cc

```diff
@@ -0,0 +1,30 @@
+#include "src/ir/selector_access_path.h"
+#include "selector.h"
+
+#include "absl/strings/str_join.h"
+
+#include <string>
+
+namespace raksha::ir {
+
+// Check the components of the two access paths for equality. This is the
+// same as their component vectors being equal according to std::vector's
+// operator==.
+bool SelectorAccessPath::operator==(const SelectorAccessPath &other) const {
```

Comment:
inline this implementation in header file itself?

Reply:
Addressed in #68